### PR TITLE
Renaming vaadin-element to vaadin-notification

### PR DESCRIPTION
**Pull request by YuriyVaadin**
Connects to https://github.com/vaadin/vaadin-core/issues/137

Renaming `vaadin-element` to `vaadin-notification` and disabling visual tests until we have real tests for the component

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-notification/1)

### CODE REVIEW

**Comment by tomivirkki** [comment]
*[vaadin-notification.html, line 50 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-notification/1#-KwpjqyuvHXy7hfyHxtP:-KwpjqyuvHXy7hfyHxtQ:bli8p7f) ([raw file](https://github.com/vaadin/vaadin-notification/blob/050756233a42e2767fbde82e849b5d8fa713bab8/vaadin-notification.html#L50)):*
> ```HTML
>        */
>       window.Vaadin = window.Vaadin || {};
>       Vaadin.VaadinNotification = VaadinNotification;
> ```

Vaadin.NotificationElement

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-notification/1)*

**Comment by limonte** [comment]
Reviewed 12 of 18 files at r1, 6 of 6 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[vaadin-notification.html, line 50 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-notification/1#-KwpjqyuvHXy7hfyHxtP:-Kx7y7VK3PSSLcuQXeiY:b-896fix) ([raw file](https://github.com/vaadin/vaadin-notification/blob/050756233a42e2767fbde82e849b5d8fa713bab8/vaadin-notification.html#L50)):*
<details><summary><i>Previously, tomivirkki (Tomi Virkki) wrote…</i></summary><blockquote>

Vaadin.NotificationElement
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-notification/1)*

**Comment by tomivirkki** [comment]
:lgtm:

---

Reviewed 12 of 18 files at r1, 6 of 6 files at r2.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-notification/1#-:-Kx8rDGsjQT39BZjOIMv:bnfp4nl)*

File: vaadin-notification.html [hunk 1]
```diff
@@ -19,20 +19,22 @@
 
   <script>
     {
-
       /**
-       * `<vaadin-element>` is a Polymer 2 element.
+       * `<vaadin-notification>` is a Polymer 2 element providing an accessible and customizable notifications (toasts).
```
Comment (jouni) [inline vaadin-notification.html]: Remove the “an” article: “—providing accessible and customizable notifications (toasts).”

Comment (limonte) [reply vaadin-notification.html]: Done.

**Comment by jouni** [comment]
Reviewed 11 of 18 files at r1, 4 of 6 files at r2, 1 of 1 files at r3, 2 of 2 files at r4.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-notification/1)*

**Pull request merged**